cobra/core/reaction.py

```diff
@@ -200,14 +198,48 @@ def lower_bound(self):
         """
         return self._lower_bound
 
+    @staticmethod
+    def _check_bounds(lb, ub):
+        if lb > ub:
+            raise ValueError(
+                "The lower bound must be less than or equal to the upper "
+                "bound ({} <= {}).".format(lb, ub))
+
+    def update_variable_bounds(self):
```

Comment:
Are we pretty sure this will play OK with the context manager? I suppose the tests all pass, so that should mean it works.

Reply:
I did not test this explicitly but I only touched one test to change from `AssertionError` to `ValueError`.